UserInfoSvcs/UserInfoManager/UserInfoManagerImpl.cpp

```diff
@@ -842,6 +846,7 @@ namespace UserInfoSvcs
   {
     static const char* FUN = "UserInfoManager::get_user_profile()";
 
+    metrics_raii raii_tmp(composite_metrics_provider_,(std::string)"UserInfoManagerImpl::get_user_profile");
```

Comment:
Please rename class name for style,
remove std::string explicit conversion (it confuses and raises questions).
Add space after ','